code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -134,72 +134,81 @@
 
 /mob/living/carbon/human/adjustCloneLoss(amount)
```

Comment:
Care to guard clause the start of this proc with `if(!amount)`? It makes it easier for us later and is logical that adjusting something by 0 takes up no more of our time.

Reply:
I can put it under the `. = ..()` line, as the parent proc returns flags related to status updates which might be relevant somewhere